config/locales/en.yml

```diff
@@ -674,12 +674,16 @@ en:
         reason: Cancellation reasons
         rejection_category:
           fully_booked: The date you requested is fully booked
-          accepted_on_ttc: We cannot offer you school experience because you have already been accepted on a Teacher Training Course
-          date_not_available: We cannot support the date you have requested
-          no_relevant_degree: We do not believe you have a relevant degree for the school experience you are applying for
-          no_phase_availability: We are unable to offer you school experience for the teaching phase you are interested in
-          candidate_not_local: We are looking for candidates that live locally to the school
+          accepted_on_ttc: We cannot offer you school experience because you've already been accepted on a teacher training course.
+          date_not_available: We can no longer offer the date you've requested.
+          no_relevant_degree: We do not believe you have a relevant degree for the school experience you're applying for.
+          no_phase_availability: We're unable to offer you school experience for the teaching phase you're interested in.
+          candidate_not_local: We're looking for candidates that live locally to the school.
           duplicate: This is a duplicate request
```

Comment:
Could we add a full stop at the end of this sentence?

Reply:
This has been fixed in #2177